DESCRIPTION

```diff
@@ -1,10 +1,11 @@
 Package: polished
 Type: Package
 Title: Authentication and Administration for 'shiny' Apps
-Version: 0.2.0.9000
+Version: 0.3.0.9000
```

Comment:
we don't want to bump to version 0.3.0 until the next CRAN release.  If you want to bump the dev version, the next version should be 0.2.0.9001.  Please update.